.link_checker_config.json

```
@@ -0,0 +1,41 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^http://scaleway.com"
+    }
+  ],
+  "replacementPatterns": [
+    {
+      "pattern": "^/.attachments",
+      "replacement": "https://scaleway.com/en/docs/.attachments"
+    },
+    {
+      "pattern": "^/",
+      "replacement": "https://scaleway.com/en/docs/"
+    },
+    {
+      "pattern": "%20",
+      "replacement": "-",
+      "global": true
+    }
+  ],
+  "httpHeaders": [
+    {
+      "urls": [
+        "https://scaleway.com/en/docs"
+      ],
+      "headers": {
+        "Authorization": "Basic Zm9vOmJhcg==",
+        "Foo": "Bar"
+      }
```

```suggestion
      ]
```

This seems to be useless ;)